### DISCUSSION TITLE
where to check up to date api documentation?

### DISCUSSION THREAD

**Discussion by atjshop** [wechaty/wechaty#2063] [Q&A] [answered]
Is this the latest and up to date api documentation https://wechaty.js.org/docs/api/?

I'm checking the message type here https://wechaty.js.org/docs/api/message#messagetype-%E2%87%92-messagetype
The Enum for the Text is MessageType.Text, but the sample code is using bot.Message.Type.Text. And Enum MessageType is not listed in the api documentation.

Thanks.

**Comment by huan** [comment, marked as answer]
Yes, the official API documentation is at <https://wechaty.js.org/docs/api/>

Please feel free to submit a PR to improve the API documentation if you find out that we have missed any parts.
